drlx-parser/pom.xml

```diff
@@ -1,47 +1,41 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>javaparser-parent</artifactId>
-        <groupId>org.drools.javaparser</groupId>
-        <version>3.3.5-SNAPSHOT</version>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-parent</artifactId>
+      <version>7.5.0-SNAPSHOT</version>
```

Comment:
@mbiarnes this change means that the update-versions script will need to be fixed.